x/avs_task/keeper/keeper.go

```diff
@@ -0,0 +1,40 @@
+package keeper
+
+import (
+	"context"
+	"fmt"
+	"github.com/ExocoreNetwork/exocore/x/avs_task/types"
+	tasktype "github.com/ExocoreNetwork/exocore/x/avs_task/types"
+	"github.com/cometbft/cometbft/libs/log"
+	"github.com/cosmos/cosmos-sdk/codec"
+	storetypes "github.com/cosmos/cosmos-sdk/store/types"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+type (
+	Keeper struct {
+		cdc       codec.BinaryCodec
+		storeKey  storetypes.StoreKey
+		avsKeeper tasktype.AvsKeeperMock
+	}
+)
+
+func NewKeeper(
+	cdc codec.BinaryCodec,
+	storeKey storetypes.StoreKey,
+	avsKeeper tasktype.AvsKeeperMock,
```

Comment:
Please change the mock to real keeper too here

Reply:
fixed